public/stylesheets/navbar.css

```diff
@@ -1,68 +1,120 @@
+@import url('https://fonts.googleapis.com/css2?family=Texturina:wght@100&display=swap');
+
 body {
     background-color: #272727;
-
+    font-family: 'Texturina', serif;
 }
 
 nav {
     display: flex;
     width: 100%;
 
 }
+
 .nav-container {
     margin-top: 10px;
     width: 100%;
     height: 50%;
     display: flex;
-    justify-content: space-around;
+    justify-content: space-between;
     align-items: center;
     background-color: #747474;
 }
 
 .logo-container {
     display: inline-block;
-    width: 100px;
+    width: 200px;
     height: 50px;
     background-repeat: no-repeat;
-    background-size: contain;
+    background-size:cover;
```

Comment:
Please add a space between the colon and the value.

Reply:
Spaces added.